lib/elixir_auth_google.ex

```diff
@@ -29,10 +29,21 @@ defmodule ElixirAuthGoogle do
     "#{Atom.to_string(s)}://#{h}:#{p}"
   end
 
+  # When deployed to Fly.io the scheme is "http" and port is "80" ...
+  # This results in an incorrect url see:
+  # github.com/dwyl/elixir-auth-google/issues/94
+  def get_baseurl_from_conn(%{host: h, req_headers: r}) do
```

Comment:
I think this will match any `Plug.Conn`? So the following clause will never execute.

Reply:
Yeah, I _want_ this to match `conn` so that I don't have to update any of my `Fly.io` apps.
Meanwhile if you keep reading there's a different function that bypasses `get_baseurl_from_conn/1` completely. 😉